### ISSUE TITLE
Spark SQL commands

### ISSUE THREAD

**Issue by amelio-vazquez-reina** [qubole/qds-sdk-py#111]
I am hoping to use this library to run Spark SQL commands on a Qubole cluster. I see some references to Spark in the code, but not in the documentation ([README.rst](https://github.com/qubole/qds-sdk-py/blob/master/README.rst)). Any guidance on this topic would be great.

**Comment by mindprince** [comment]
Hey,

We just published v1.6.2. It contains support for submitting Spark SQL queries from the SDK.
```
Usage: sparkcmd <submit|run> [options]

Options:
  -h, --help            show this help message and exit
  --cmdline=CMDLINE     command line for Spark
  --sql=SQL             sql for Spark
  -f SCRIPT_LOCATION, --script_location=SCRIPT_LOCATION
                        Path where spark program to run is stored. Has to be a
                        local file path
  --macros=MACROS       expressions to expand macros used in query
  --tags=TAGS           comma-separated list of tags to be associated with the
                        query ( e.g., tag1 tag1,tag2 )
  --cluster-label=LABEL
                        the label of the cluster to run the command on
  --notify              sends an email on command completion
  --name=NAME           Assign a name to this query
  --arguments=ARGUMENTS
                        Spark Submit Command Line Options
  --user_program_arguments=USER_PROGRAM_ARGUMENTS
                        Arguments for User Program
  --print-logs          Fetch logs and print them to stderr.
```

So, you can do something like the following:
```
qds.py sparkcmd submit --sql 'YOUR QUERY' --cluster-label SPARK-CLUSTER-LABEL
```

**Comment by amelio-vazquez-reina** [comment]
Thanks @mindprince That's great. I will give this a try soon.
